### ISSUE TITLE
[BUG]: Snackbar doesn't show up when searching by flag

### ISSUE THREAD

**Issue by brishtibheja** [ankidroid/Anki-Android#16006]
### Checked for duplicates?

- [X] This issue is not a duplicate

### Does it also happen in the desktop version?

- [ ] This bug does not occur in the latest version of Anki

### What are the steps to reproduce this bug?

1. Go to card browser
2. Search cards by flag from the menu
3. click on a card
4. Do another search

### Expected behaviour

snackbar shows number of cards

### Actual behaviour

snackbar doesn't appear. If you click on search bar and press enter then it appears this time.

### Debug info

```text
AnkiDroid Version = 2.17.5 (1c1aa94ec466f37d33fd2f75020d9f9021ac811e)

Backend Version = 0.1.34-anki23.12.1 (23.12.1 1a1d4d5419c6b57ef3baf99c9d2d9cf85d36ae0a)

Android Version = 11 (SDK 30)

ProductFlavor = full

Manufacturer = realme

Model = RMX2193

Hardware = mt6768

Webview User Agent = Mozilla/5.0 (Linux; Android 11; RMX2193 Build/RP1A.200720.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/122.0.6261.119 Mobile Safari/537.36

ACRA UUID = cf265e01-9693-407f-984d-61ddf9698fac

Crash Reports Enabled = true
```

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by neeldoshii** [comment]
I am unable to reproduce the issue. Can you tell me am I wrong somewhere.

https://github.com/ankidroid/Anki-Android/assets/60827173/e23af1c8-fada-4338-bacc-d9f1dd2a44dd

If reproduced I like to work on this.

**Comment by brishtibheja** [comment]
Yes I noticed I can't reproduce this in the latest alpha. Was this fixed somehow? @david-allison

**Comment by david-allison** [comment]
Likely flaky, not fixed

**Comment by brishtibheja** [comment]
@neeldoshii If you are still interested, I have updated the issue. You need to click on a card and then need to do another search. Let me know if you can't reproduce it and I will see If I've missed something.
